### PR TITLE
Do not change color of a meta tag

### PR DESCRIPTION
**Pull request by victorchan314**
#268

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -337,9 +336,16 @@ def create(self, request, **kwargs):
                         },
                         status=400,
                     )
-                # the post m2m hook will add tag
-                puzzle.metas.add(meta)
+                else:
```
Comment (asdfryan) [inline api/views.py:339]: This else is not necessary, no?

Comment (victorchan314) [reply api/views.py:339]: No, but I think an explicit else statement is easier to read.

Comment (asdfryan) [reply api/views.py:339]: Fair enough.

**Review by asdfryan — APPROVED** [review]

**Comment by asdfryan** [comment]
Looks like black is showing diffs, but otherwise LGTM

**Review by rgossiaux — CHANGES_REQUESTED** [review]
Don't actually have any feedback (haven't read it) but since we're code freezing we shouldn't merge this until after the hunt

**Review by rgossiaux — APPROVED** [review]
Thanks! (belatedly)

**Pull request merged**